src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -579,12 +612,29 @@ export default class PhotoField extends React.Component {
     this.setState(dragActive);
   }
 
+  updatePreviewSrc = (src) => {
+    this.setState({ previewSrc: src });
+  }
+
   render() {
     const { formData, formContext } = this.props;
     const file = formData || {};
     const onReview = formContext.reviewMode;
 
-    if (onReview) return <PhotoReviewDescription url={getImageUrl(file)}/>;
+    if (onReview) {
```

Comment:
What is `reviewMode`?

Reply:
On the review page, we render every field in a read only view. Normally there are separate widgets for that (the SchemaForm component choose between them depending on a prop), but for custom fields it's easier to switch between them in the Field component.